src/components/dapp-staking-v2/StakingTop.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="wrapper-main">
+    <TopMetric />
+
+    <div class="divider"></div>
+
+    <MyStaking />
+
+    <div class="divider"></div>
+
+    <DappList category="De-Fi" />
+
+    <AdsArea />
+
+    <div class="divider"></div>
+
+    <DappList category="Infra" />
+
+    <div class="divider"></div>
+
+    <DappList category="NFT" />
+
+    <Teleport to="#app--main">
+      <div :class="!isLoading && 'highest-z-index'">
+        <!-- <ModalRegisterDapp
+          v-if="showRegisterDappModal"
+          v-model:is-open="showRegisterDappModal"
+          :show-close-button="false"
+        /> -->
+      </div>
+    </Teleport>
+  </div>
+</template>
+
+<script lang="ts">
+import { useMeta } from 'quasar';
+import { useStore } from 'src/store';
+import TopMetric from 'src/components/dapp-staking-v2/my-staking/TopMetric.vue';
+import MyStaking from 'src/components/dapp-staking-v2/my-staking/MyStaking.vue';
+import DappList from 'src/components/dapp-staking-v2/my-staking/DappList.vue';
+import { computed, defineComponent } from 'vue';
+import AdsArea from './my-staking/AdsArea.vue';
+
+export default defineComponent({
+  components: {
+    TopMetric,
+    MyStaking,
+    DappList,
+    AdsArea,
+  },
+  setup() {
+    useMeta({ title: 'Discover dApps' });
+    const store = useStore();
+    const isLoading = computed<boolean>(() => store.getters['general/isLoading']);
+
+    return {
+      isLoading,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@import 'src/css/quasar.variables.scss';
+
+.wrapper-main {
+  padding: 0px 0px 24px 0px;
+  margin: 0 auto;
+
+  @media (max-width: $lg) {
```

Comment:
@sirius651  We should use `min-width` instead of `max-width` to make the code consistent. Let's fix it later.
CC: @bobo-k2

Reply:
we should make guideline for all of this

Reply:
`max-width` is a desktop first approach. To build a responsive website, we should start with mobile first, then adjust the width bit by bit. Otherwise, styling will be broken easily.  

Ref:
https://nikitahl.com/difference-between-min-width-vs-max-width

https://stackoverflow.com/questions/16647380/max-width-vs-min-width

https://dev.to/savvasstephnds/building-a-responsive-website-start-with-mobile-first-ho2